additional screenshots

johndoh opened the pull request:
- Add a couple of screenshots of the different mail view layouts (larry and classic only) and the login screen for completeness. These already existed just not included in the display
- Add a simple cache buster based on when the screenshots where generated (this is just a bit of housekeeping)

pabzm commented:
Thank you for noticing this after that much time and doing the cores!

Do you happen to know if there maybe was some intention in leaving them out previously? As far as I can tell you committed the images and the overview changes to the repository yourself, but I don't know if there was communication around it.

johndoh commented:
I built a new layout for the website but at the time I kept all the content the same as the existing site. At that time the site only had screenshots of the Larry skin (the default skin at the time). I refreshed those images and added similar ones for the Classic skin. I do not know how the original choice for what to include was made.

Somewhere along the way I extended the script I use to make the screenshots and added 2 or 3 more to the collection. I was thinking either those additional screenshots could be used or removed from the collection. I’m not sure at all if we need a screenshot of the login screen, there is not much to see. Its only there for completeness. The ones of the additional mailbox views at least show more of what those skins can do.

This PR makes the decision, either we use them or scrap them.

pabzm commented:
Thank you for the explanation!

I was wondering about the login screenshot, too. How about we leave that one out and use integrate only the others? (From my point of view we could leave the images in the repo, but if you think we should remove them, that's also fine with me.)

pabzm commented:
Thank you very much!
